pkg/storage/config/schema_config.go

```diff
@@ -371,6 +373,42 @@ func (cfg *PeriodConfig) applyDefaults() {
 	}
 }
 
+// ChunkFormat returns chunk format including it's headBlockFormat corresponding to the `schema` version
+// in the given `PeriodConfig`.
+func (cfg *PeriodConfig) ChunkFormat() (byte, chunkenc.HeadBlockFmt, error) {
+	sver, err := cfg.VersionAsInt()
+	if err != nil {
+		return 0, 0, fmt.Errorf("failed to get chunk format: %w", err)
+	}
+
+	switch {
+	case sver <= 12:
+		return chunkenc.ChunkFormatV3, chunkenc.UnorderedHeadBlockFmt, nil
+	case sver == 13:
+		fallthrough
+	default:
+		return chunkenc.ChunkFormatV4, chunkenc.UnorderedWithNonIndexedLabelsHeadBlockFmt, nil
+	}
```

Comment:
This logic partially duplicates `pkg/chunkenc/memchunk.go:416-425`

I think it would be great to have a single source of truth for the binding of chunk format to head block format.

Reply:
Good idea. Done.